core/src/main/java/com/github/mizool/core/converter/EnumConverter.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2018 incub8 Software Labs GmbH
+ * Copyright 2018 protel Hotelsoftware GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.mizool.core.converter;
+
+public class EnumConverter
+{
+    public String fromPojo(Enum<?> enumValue)
+    {
+        return enumValue.name();
+    }
+
+    public <T extends Enum<T>> T toPojo(Class<T> enumClass, String value)
+    {
+        return T.valueOf(enumClass, value);
```

Comment:
Won't this cause a compiler warning? maybe the Mizool maven build does not show them yet, but I think the IDE would.

Anyway, I would prefer `Enum.valueOf(enumClass, value);` - it will return a `T` as well.